### DISCUSSION TITLE
Display options

### DISCUSSION THREAD

**Discussion by phirestalker** [giongto35/cloud-game#398] [General]
I was thinking about deploying this on my local network. I had set up a pi 3b for gaming, but there are cores missing because of armv7 cpu.

I want to know if I can play in full-screen mode without the side buttons on my instance. If so, how can I configure it to not show them by default?

**Comment by sergystepanov** [comment]
You can press the "F" button on the keyboard or rebind this key in the options. Right now it is not possible to change the UI, maybe, in the future updates with the DOSBox/PC keyboard support.

**Reply by phirestalker** [reply]
How will I know when this is implemented?

**Reply by sergystepanov** [reply]
The changes will be in the commit log and on cloudretro.io for testing.

**Comment by sergystepanov** [comment]
Auto full-screen mode has been added in [a349fdd](https://github.com/giongto35/cloud-game/commit/a349fdd0cf99036b45d925e2a0805ddb8756cb25).
